src/reducers/query.jsx

```diff
@@ -53,7 +68,20 @@ export function toggleFilter(state, action) {
   }
 }
 
-// TODO: Set defaultQueryState to recognize existing URL params
+function removeFilter(state, action) {
```

Comment:
FILTER_CHANGE also does pill removal in a way (detects if already present then filters from the returned array, rather than splicing like this approach). For future readability, do you think we should differentiate when / how to use filter removal / filter change or are comments the best venue for now?